src/base/Image.php

```diff
@@ -74,7 +74,7 @@ abstract public function crop(int $x1, int $x2, int $y1, int $y2): self;
      * @param bool $scaleIfSmaller
      * @return static Self reference
      */
-    abstract public function scaleToFit(?int $targetWidth, ?int $targetHeight, bool $scaleIfSmaller = true): self;
+    abstract public function scaleToFit(?int $targetWidth, ?int $targetHeight, bool $scaleIfSmaller = true, string $position = 'center-center'): self;
```

Comment:
Missing `@param` for `$position`

Reply:
Also if the fill stuff is exclusive to `Raster`, it feels weird to have `$position` defined here in the base class. Maybe `Raster` should just be defining its own `scaleToFitAndFill()` method that combines `scaleToFit()` and `setFill()`?